azure-pipelines.yml

```diff
@@ -3,6 +3,7 @@ trigger:
     include:
     - main
     - feature/*
+    - features/*
```

Comment:
This feels like it's working around people who misname their branches. Is it really necessary?